client/my-sites/media/main.jsx

```diff
@@ -167,13 +167,21 @@ class Media extends Component {
 			return null;
 		}
 
-		const { translate } = this.props;
+		const { selectedSite, translate } = this.props;
+		const selectedMediaCount = MediaLibrarySelectedStore.getAll( selectedSite.ID ).length;
```

Comment:
Could we maybe use [`getSelectedItems`](https://github.com/Automattic/wp-calypso/blob/3f78dc9442de6c63a92f5ad290ba9c589100c9c2/client/my-sites/media/main.jsx#L334) here?

Reply:
Would it be better? I took this approach because it's identical to the one used later in the file, would that be worth changing too?

https://github.com/Automattic/wp-calypso/blob/3f78dc9442de6c63a92f5ad290ba9c589100c9c2/client/my-sites/media/main.jsx#L247-L248

Reply:
On a second look it might be ok the current approach. Other usages of `getSelectedItems` end up falling back to `MediaLibrarySelectedStore`

https://github.com/Automattic/wp-calypso/blob/3f78dc9442de6c63a92f5ad290ba9c589100c9c2/client/my-sites/media/main.jsx#L314-L319